src/main/resources/application-form-flow-library.yaml

```diff
@@ -1,4 +0,0 @@
-spring:
-  flyway:
-    placeholders:
-      uuid_function: "gen_random_uuid"
```

Comment:
Taking this out means that V4 migration will not work. I think we need to update the V4 migration as well (just to remove the variable), I wouldn't change anything else.